runtime/parachains/src/paras_inherent/tests.rs

```diff
@@ -28,6 +28,7 @@ mod enter {
 	};
 	use assert_matches::assert_matches;
 	use frame_support::assert_ok;
+	use sp_runtime::ModuleError;
```

Comment:
These tests are only executed when `runtime-benchmarks` are not enabled. This means that these tests are never running in CI. This needs to be fixed!

Reply:
(see `mod enter`)

Reply:
These tests here would fail if we run with feature benchmark, so I added an extra `cargo test` for the `polkadot-runtime-parachains`